robottelo/hosts.py

```diff
@@ -1395,6 +1395,7 @@ def __init__(self, hostname=None, **kwargs):
         from robottelo.config import settings
 
         hostname = hostname or settings.server.hostname  # instance attr set by broker.Host
+        self.version = kwargs.get('version', settings.server.version.release)
```

Comment:
because this is running on the RHEL8 machine where we are trying to install Satellite. we will not be having the satellite rpm package and `upstream` is incorrect for me. This was before added support for #10079. Maybe I can pass the sat version directly from the test. But it has to be from the setting object only.

Reply:
It seems like the most correct way to get the version for this specific test.